devtools/minimal.txt

```diff
@@ -8,3 +8,4 @@ matplotlib
 svgwrite
 ujson
 psutil
+mdtraj
```

Comment:
due to the move `mdtraj` became a hard requirement

Reply:
Can't you do a `try/except ImportError` and then use `paths.integration_tools.error_if_no_mdtraj('MDTrajTopology')` at the start of the `from_dict`? (Maybe also `__init__`, although `__init__` doesn't actually use MDTraj.